src/erl.rs

```diff
@@ -1163,14 +1187,14 @@ fn case<'a>(subjects: &'a [TypedExpr], cs: &'a [TypedClause], env: &mut Env<'a>)
     } else {
         tuple(subjects.iter().map(|e| maybe_block_expr(e, env)))
     };
-    "case "
-        .to_doc()
-        .append(subjects_doc)
-        .append(" of")
-        .append(line().append(clauses(cs, env)).nest(INDENT))
-        .append(line())
-        .append("end")
-        .group()
+    docvec![
+        "case ",
+        subjects_doc,
+        " of",
+        docvec![line(), clauses(cs, env, location)].nest(INDENT),
+        line(),
+        "end",
+    ]
```

Comment:
I think we lost a `group` here?

Reply:
> 
> 
> I think we lost a `group` here?

Sorry about that, fixed!
Just curious, I believe the `group()` function is used to create a `Group` data type that would be used to format the indents? I wonder why it did not make a difference in the unit tests when I left it out 🤔 

Also, CI seems to require me to run cargo fmt on `src/project/tests.rs` on the entire file, even in the places where I have not made changes, hope that is ok

Reply:
No worries, easy to do if the tests don't catch it. It might not be needed now I think about it, but let's keep it in case there's a hole in the tests. We can remove it later in another PR if we want.

Formatting sounds good, thanks